### PR TITLE
Cleaning up spacing, typos, incorrect heading levels, etc...

### PR DESCRIPTION
**Pull request by lnewson**
Related Ticket: N/A

Description of Changes:
* This fixes a number of issues found when spiking out converting to AsciiDoc, as it has more stringent level checks and builds the nav based on the actual content. The conversion also highlighted we're missing some blank lines to separate content where appropriate and there's also a number of other fixes I manually noticed.

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: _includes/codepens/full-featured/index.js [hunk 1]
```diff
@@ -273,7 +273,7 @@ tinymce.init({
   spellchecker_whitelist: ['Ephox', 'Moxiecode'],
   tinycomments_mode: 'embedded',
   content_style: ".mymention{ color: gray; }",
-  contextmenu: "link image imagetools table configurepermanentpen",
+  contextmenu: "link image imagetools table spellchecker configurepermanentpen",
```
Comment (lnewson) [inline _includes/codepens/full-featured/index.js:276]: We use as you type, but don't have the context menu enabled so you can't fix typos in the demo.

File: _includes/release-notes/bugfixes.md [hunk 2]
```diff
@@ -1,502 +0,0 @@
-The community was invited to test the TinyMCE 5.0 editor and report bugs and issues in the Developers Preview release. The following issues have been fixed in the TinyMCE 5.0 Beta version.
```
Comment (lnewson) [inline _includes/release-notes/bugfixes.md:1]: This was leftover from the beta, so I removed it.

File: advanced/creating-a-skin.md [hunk 3]
```diff
@@ -98,7 +98,7 @@ Your skin.less file should now look like this:
 ```
 Switch to the web browser. Select your skin from the *Skin menu*. It should show a fiery red editor
 
-![**{{site.productname}} skin SDK for Silver theme**]({{site.baseurl}}/images/SDKforsilverCustomExample.png)
```
Comment (lnewson) [inline advanced/creating-a-skin.md:101]: This formats incorrectly, as you can't have bold alt text so it actually includes the asterisks verbatim.

File: advanced/editor-command-identifiers.md [hunk 4]
```diff
@@ -76,57 +76,57 @@ The commands on the following table are provided by the {{site.productname}} edi
...
+tinymce.activeEditor.execCommand('Bold');
+tinymce.activeEditor.execCommand('Italic');
+tinymce.activeEditor.execCommand('Underline');
+tinymce.activeEditor.execCommand('Strikethrough');
+tinymce.activeEditor.execCommand('Superscript');
+tinymce.activeEditor.execCommand('Subscript');
+tinymce.activeEditor.execCommand('Cut');
+tinymce.activeEditor.execCommand('Copy');
+tinymce.activeEditor.execCommand('Paste');
+tinymce.activeEditor.execCommand('mceInsertLink', false, 'https://www.tiny.cloud');
```
Comment (lnewson) [inline advanced/editor-command-identifiers.md:88]: Missing comma here :disappointed:

File: changelog.md [hunk 5]
```diff
@@ -566,7 +566,7 @@ class: changelog
 * Added a new `addNestedMenuItem()` UI registry function and changed all nested menu items to use the new registry functions.
 * Added `title` attribute to color swatch colors.
 * Added `anchorbar` component to anchor inline toolbar dialogs to instead of the toolbar.
-* Added support for **toolbar<n>** and **toolbar array** config options to be squashed into a single toolbar and not create multiple toolbars.
```
Comment (lnewson) [inline changelog.md:569]: This was rendering as a literal `<n>` element.

File: general-configuration-guide/multiple-editors.md [hunk 6]
```diff
@@ -27,10 +27,10 @@ The following example breaks the page into two separate editable areas. Each are
 
 <body>
   <form method="post">
-    <h1>Multple editors on a page: Section 1</h1>
+    <h1>Multiple editors on a page: Section 1</h1>
```
Comment (lnewson) [inline general-configuration-guide/multiple-editors.md:30]: Typos

File: plugins/casechange.md [hunk 7]
```diff
@@ -46,12 +46,10 @@ tinymce.init({
 ```
 
 
-### Types of Formats
+## Types of Formats
 
 Currently, there are three types of formats available:
 
-![**Case Change**]({{site.baseurl}}/images/casechange.png)
```
Comment (lnewson) [inline plugins/casechange.md:53]: This image was deleted back in 63df78e0f87f798adbb988b43bb076eeba4a2cd5 and the reference in the enterprise page was removed, but we missed this one.

File: plugins/insertdatetime.md [hunk 8]
```diff
@@ -37,7 +37,7 @@ tinymce.init({
   selector: "textarea",  // change this value according to your HTML
   plugins: "insertdatetime",
   menubar: "insert",
-  toolbar: "insertdatetime"
+  toolbar: "insertdatetime",
```
Comment (lnewson) [inline plugins/insertdatetime.md:40]: Broken code examples because of the missing comma.

File: plugins/table.md [hunk 9]
```diff
@@ -438,13 +444,13 @@ tinymce.init({
...
 
-### Events
+## Events
 
 | Name | Description |
 |------| ----------- |
 | newrow | Fired when a row is created |
 | newcell | Fired when a cell is created |
-| ObectResizeStart | Fired when a resize action is started on a table, row, column or cell using the resize bars |
+| ObjectResizeStart | Fired when a resize action is started on a table, row, column or cell using the resize bars |
```
Comment (lnewson) [inline plugins/table.md:453]: Typo in event name

File: ui-components/autocompleter.md [hunk 10]
```diff
@@ -26,10 +26,10 @@ The two arguments this method take are:
 | Name | Value | Requirement | Description |
 | ---- | ----- | ----------- | ----------- |
 | ch | string | Required | The character to trigger the autocompleter. |
-| fetch |  (pattern: string, maxResults: number, fetchOptions: Record<string, any>) => Promise<AutocompleterItem[]> | Required | A function that is passed the current matched text pattern, the maximum number of expected results and any additional fetch options. The function should return a promise containing matching results. |
-| onAction | (api, rng: Range, value: string) => void | Required | A function invoked when a fetched item is selected. |
+| fetch | `(pattern: string, maxResults: number, fetchOptions: Record<string, any>) => Promise<AutocompleterItem[]>` | Required | A function that is passed the current matched text pattern, the maximum number of expected results and any additional fetch options. The function should return a promise containing matching results. |
```
Comment (lnewson) [inline ui-components/autocompleter.md:29]: This was inconsistent with how we format all other function types, so added backticks to make it consistent.

File: ui-components/contextform.md [hunk 11]
```diff
@@ -49,49 +49,49 @@ This relates to the form itself. The form specifications are:
 
 {% include context/positioning.md %}
 
-### Context form buttons
+## Context form buttons
 
 Unlike normal context toolbar buttons, Context form buttons are not registered beforehand. Instead, you need to define each button completely in the `commands` section.
 
-#### contextformbutton
+### Context form button
```
Comment (lnewson) [inline ui-components/contextform.md:56]: The title was inconsistent with the below ` Context form toggle button` heading.

File: _includes/configuration/urlconverter-callback.md [hunk 12]
```diff
@@ -9,7 +9,7 @@ The format of the converter function is: `URLConverter(url, node, on_save)`, whe
 * on_save is always set to true.
 * name = the attribute name that is being set.
 
-This function should return the converted URL as a string. This option is set to an internal {{site.productname}} function <editor>.[convertURL()]({{ site.baseurl }}/api/tinymce/tinymce.editor/) by default. You may call this function from your extension in order to use the built-in convert options.
+This function should return the converted URL as a string. This option is set to an internal {{site.productname}} function [convertURL()]({{ site.baseurl }}/api/tinymce/tinymce.editor/) by default. You may call this function from your extension in order to use the built-in convert options.
```
Comment (lnewson) [inline _includes/configuration/urlconverter-callback.md:12]: `<editor>` again was being rendered as an actual element and didn't seem to be adding value anyways, so I just removed it.

File: advanced/editor-command-identifiers.md (hunk 4 above)
Comment (tylerkelly13) [reply advanced/editor-command-identifiers.md:88]: :disappointed:

File: plugins/casechange.md (hunk 7 above)
Comment (tylerkelly13) [reply plugins/casechange.md:53]: actually, this one is kind-of funny to see rendered

File: enterprise/server/index.md [hunk 13]
```diff
@@ -109,16 +115,19 @@ For other versions of Tomcat on Windows, check the Tomcat documentation for that
...
 
 For other versions of Jetty on Windows, check the Jetty documentation for that version.
 
 ###### As a Windows service
 
 Follow the instructions in [Startup via Windows Service](https://www.eclipse.org/jetty/documentation/current/startup-windows-service.html) for Jetty 9.4.5. Remember to append the following snippet to the line beginning with `set PR_JVMOPTIONS` in your `install-jetty-service.bat` script:
 
-    ;-Dephox.config.file="C:\config\file\location\application.conf"
+```
+;-Dephox.config.file="C:\config\file\location\application.conf"
```
Comment (EkimChau) [inline enterprise/server/index.md:129]: ^ that command doesn't look right

File: enterprise/server/index.md [hunk 14]
```diff
@@ -109,16 +115,19 @@ For other versions of Tomcat on Windows, check the Tomcat documentation for that
 
 If you're following the instructions for [Starting Jetty](https://www.eclipse.org/jetty/documentation/current/startup.html) for Jetty 9.4.5, the path to the configuration file can simply be supplied as a command option:
 
-    java -D"ephox.config.file=C:\config\file\location\application.conf" -jar C:\jetty\install\directory\start.jar
-
+```
+java -D"ephox.config.file=C:\config\file\location\application.conf" -jar C:\jetty\install\directory\start.jar
```
Comment (EkimChau) [inline enterprise/server/index.md:119]: should there be a space after -D ?

File: enterprise/server/index.md [hunk 15]
```diff
@@ -144,13 +153,17 @@ If you've obtained Tomcat from your distribution's package manager, refer to you
 
 Create or edit the script `/opt/tomcat/bin/setenv.sh` to contain the following line:
 
-    CATALINA_OPTS=" -Dephox.config.file=/config/file/location/application.conf"
+```
+CATALINA_OPTS=" -Dephox.config.file=/config/file/location/application.conf"
```
Comment (EkimChau) [inline enterprise/server/index.md:157]: please confirm  -Dephox.config is correct, theres a few of them

File: enterprise/server/index.md (hunk 13 above)
Comment (tylerkelly13) [reply enterprise/server/index.md:129]: apparently these are correct, although I need to review this content as some point

File: enterprise/server/index.md (hunk 14 above)
Comment (tylerkelly13) [reply enterprise/server/index.md:119]: Nope.

File: enterprise/server/index.md (hunk 15 above)
Comment (tylerkelly13) [reply enterprise/server/index.md:157]: Nope. this is correct.

File: enterprise/server/index.md (hunk 14 above)
Comment (lnewson) [reply enterprise/server/index.md:119]: See https://docs.oracle.com/en/java/javase/13/docs/specs/man/java.html#standard-options-for-java

**Pull request merged**